cmd/git-sync/main.go

```diff
@@ -283,6 +286,13 @@ func main() {
 		}
 	}
 
+	if *flGitConfigs != "" {
```

Comment:
Comment here that this should be the very last config section, so it overrides all others.